app.cfg.dist

```diff
@@ -1,3 +1,6 @@
+[db]
+uri = mysql+pymysql://username:password@hostname/profiles
```

Comment:
any reason for MySQL vs Postgresql (used in many of our Python projects)?

Reply:
I don't think it matters from the code perspective.

Reply:
@lsh-0 any preference for an [dev]ops perspective?

Reply:
if it doesn't affect the code, then postgresql please.